sushi-config.yaml

```diff
@@ -116,20 +116,20 @@ pages:
     title: Test Plan
     generation: markdown
   issues.md:
-    title: Significant Changes and Issues
+    title: Significant Changes & Issues
     generation: markdown
   other.md:
-    title: Changes to Other Documents
+    title: Changes to Other IHE Documents
     generation: markdown
   download.md:
-    title: Download and analysis
+    title: Download and Analysis
     generation: markdown
 
 menu:
   IRA Home: index.html
   Volume 1:
     Introduction: volume-1.html
-    Actors and Transactions: volume-1.html#1531-integrating-reporting-applications
+    Actors and Transactions: volume-1.html#1531-ira-actors-transactions-and-content-modules
```

Comment:
I made this change to correct the link that matches the change in the section title.